### PR TITLE
Develop anchortab fix

### PR DESCRIPTION
**Pull request by jleong01**
- Refactored AnchorTab Component
- Introduced Title Bean
- Removed old Bean 
- Updated JS calculation
- Removed Padding for title component
- Updated AnchorTab component to loop through all child

### CODE REVIEW

File: core/src/main/java/com/taylorsuniversity/models/bean/TitleModelBean.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.models.annotations.Model;
+import org.apache.sling.models.annotations.Optional;
+
+import javax.inject.Inject;
+
+/**
+ * The Class TitleModelBean.
+ */
+@Model(adaptables = Resource.class)
```
Comment (bsinghal1) [inline core/src/main/java/com/taylorsuniversity/models/bean/TitleModelBean.java]: Please remove the Resource annotation. Please use the constructor instead.

File: ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/content/title/title.html [hunk 2]
```diff
@@ -4,12 +4,10 @@
...
-            <div class="black-header-with-text t-pt t-px">
-	            <div class="bg-black d-inline-block py-2 px-4">
-	                <h2 class="d-inline text-white">${properties.title}</h2>
-	            </div>
-	            <p class="py-3">${properties.description}</p>
-	        </div>
-     </div>
+        <div class="black-header-with-text t-pt">
+            <div class="bg-black d-inline-block py-2 px-4">
+                <h2 class="d-inline text-white">${properties.title @ i18n}</h2>
```
Comment (bsinghal1) [inline ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/content/title/title.html]: Please remove @ i18n from here.

File: ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/content/title/title.html [hunk 3]
```diff
@@ -4,12 +4,10 @@
...
+            </div>
+            <p class="py-3">${properties.description @ i18n}</p>
```
Comment (bsinghal1) [inline ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/content/title/title.html]: Please remove @ i18n from here.

File: ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/content/title/title.html (hunk 2 above)
Comment (jleong01) [reply ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/content/title/title.html]: removed via merge from develop branch

File: ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/content/title/title.html (hunk 3 above)
Comment (jleong01) [reply ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/content/title/title.html]: removed via merge from develop branch

File: core/src/main/java/com/taylorsuniversity/models/bean/TitleModelBean.java (hunk 1 above)
Comment (jleong01) [reply core/src/main/java/com/taylorsuniversity/models/bean/TitleModelBean.java]: changed to constructor

File: ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/content/title/title.html (hunk 3 above)
Comment (jleong01) [reply ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/content/title/title.html]: removed i18n

File: ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/content/title/title.html (hunk 2 above)
Comment (jleong01) [reply ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/content/title/title.html]: removed i18n

**Pull request merged**